### DISCUSSION TITLE
Random select make lags

### DISCUSSION THREAD

**Discussion by Dominilk01** [ppy/osu#28036] [Q&A]
### Type

Performance

### Bug description

Weird lags on random select

### Screenshots or videos

https://github.com/ppy/osu/assets/108392199/fff04e94-d7a6-4910-aebd-3c3ad697c982

### Version

2024.412-1-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15155383/compressed-logs.zip)

**Comment by BlockHead007** [comment]
That is probably because game loads all banners of beatmaps nearby selected one. I do experience same thing but it is much worse for me because my game is located on hdd and there is about 15k beatmap sets.

**Comment by Dominilk01** [comment]
But I didn't had the lags before playing on multi
